DynamicAnalyzer/views/android/environment.py

```diff
@@ -95,17 +117,26 @@ def configure_proxy(self, project):
 
     def install_mobsf_ca(self, action):
         """Install or Remove MobSF Root CA."""
+        ca_construct = '{}.0'
+        pem = open(get_ca_dir(), 'rb').read()
+        ca_file = crypto.load_certificate(crypto.FILETYPE_PEM, pem)
+        ca_file_hash = hex(ca_file.subject_name_hash()).lstrip('0x')
         ca_file = os.path.join('/system/etc/security/cacerts/',
-                               settings.ROOT_CA)
+                               ca_construct.format(ca_file_hash))
         if action == 'install':
             logger.info('Installing MobSF RootCA')
-            self.adb_command(['push', get_ca_dir(), ca_file])
+            self.adb_command(['push',
+                              get_ca_dir(),
+                              ca_file])
             self.adb_command(['chmod',
                               '644',
                               ca_file], True)
         elif action == 'remove':
             logger.info('Removing MobSF RootCA')
-            self.adb_command(['rm', ca_file], True)
+            self.adb_command(['rm',
+                              ca_file], True)
+        # TODO: determine if an 'adb restart' is needed
```

Comment:
Remove this comment if not required

Reply:
Not required any more, will be addressed in the next commits.